src/Microsoft.AspNetCore.Mvc.RazorPages/ApplicationModels/IPagePropertyModelConvention.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.ApplicationModels
+{
+    /// <summary>
+    /// Allows customization of the <see cref="PagePropertyModel"/>.
+    /// </summary>
+    public interface IPagePropertyModelConvention
```

Comment:
We don't an equivalent for controllers. I think there's value in having them since they're analogous to parameters.

Reply:
I don't think we should have one of these without adding it to controllers also. We should expose `PropertyModel` from `PageModel` instead

Reply:
We do have a `HandlerProperties` hanging off of `PageApplicationModel` https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.RazorPages/ApplicationModels/PageApplicationModel.cs#L116 thematically named after `ControllerProperties` that hang off of `ControllerModel`. I can add the ones in the controller too, seems like a small-ish effort

Reply:
We shouldn't have a convention for this. They are all `ParameterModel`.

Reply:
There's a separate type though. `ParameterModel` references `ActionModel` -> `ControllerModel` -> `ApplicationModel` so it's hard for it to be used in the context of a Page.`PagePropertyModel` (which already exists) references `PageApplicationModel` and is distinct from `ParameterModel` \ `PropertyModel`

Reply:
Ah I see, yeah we made a mistake with that.